src/components/Signup/ShowNewPod.jsx

```diff
@@ -0,0 +1,39 @@
+// React Imports
```

Comment:
I created a test file for this component but there is a filed bug #420  which was throwing an error on the test. I'm decided to not address it now.